client/src/main/java/org/dst/client/cmd/ClientResult.java

```diff
@@ -1,7 +1,7 @@
 package org.dst.client.cmd;
 
 public class ClientResult {
-  public String result;
+  public String result = "command is empty";
```

Comment:
Why add the default value?

Reply:
I think if there is no command parsed, we should return a default result.

Reply:
I remove this default value.